plugins/extra/noaa.go

```diff
@@ -11,26 +11,86 @@ import (
 	"unicode"
 
 	seabird "github.com/belak/go-seabird"
+	"github.com/go-xorm/xorm"
```

Comment:
Nit: in other plugins, I try to keep the order: stdlib, external, core. xorm would fall under external, similar to how it's organized in forecast.go.